.github/workflows/test-results.yml

```diff
@@ -25,10 +25,10 @@ jobs:
 
           artifacts_url=${{ github.event.workflow_run.artifacts_url }}
 
-          gh api --paginate "$artifacts_url" -q '.artifacts[] | [.name, .archive_download_url] | @tsv' | while read artifact
+          gh api --paginate "$artifacts_url" -q '.artifacts[] | [.name, .archive_download_url] | @tsv' | while read -r artifact
```

Comment:
The linter also complained about the missing `-r` flag for calls to the `read` function of Bash.  I was not quite sure whether the flag might break something but the test results were published as intended.  If something should go wrong after this PR was merged, the newly added `-r` flag is most likely to be the exact reason.

Reply:
Okay, everything seems to be fine.